sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/email/html/SendmailHeader.java

```diff
@@ -0,0 +1,41 @@
+package life.qbic.samplenotificator.components.email.html;
+
+import life.qbic.business.notification.send.NotificationEmail;
+
+/**
+ * <p>A representation of the email header necessary for the sendmail command</p>
+ */
+public class SendmailHeader {
+  String from = "noreply@qbic.life";
+  String subject = "Sample Status Update";
```

Comment:
I am missing the project code here

Reply:
We do not have information about the email content in the header. It is only used for formatting and containing the values.

Reply:
I would rather prefer not having a value here and require the header constructor to provide the subject. so we would not need to add the code here but show that this is not the actual value